### PR TITLE
Ignore non-amateur frequency from Repeaterbook (instead of failing)

### PR DESCRIPTION
**Pull request by jancona**
Evidently Repeaterbook can return repeaters with non-amateur frequencies, which was causing dzcb to fail. This PR instead skips that repeater and logs a warning.

### CODE REVIEW

**Review by masenf — APPROVED** [review]
Looks reasonable from here. Thanks for the patch.

**Comment by jancona** [comment]
Thanks for approving it. I'm seeing a "2 workflows awaiting approval" message. I assume you'll merge this when you're ready and then the workflows will run?

**Comment by masenf** [comment]
@jancona thanks for your contribution. I approved earlier today, but I didn't have a chance to run the workflows locally and merge/release until just now.

Your fix is released in [`dzcb-0.3.8`](https://github.com/mycodeplug/dzcb/releases/tag/0.3.8): https://pypi.org/project/dzcb/0.3.8/

**Pull request merged**
